zoom_python_client/utils/typed_dict_parameters.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+from typing_extensions import TypedDict
+
+logger = logging.getLogger("python_zoom_client")
+
+
+class DataType(TypedDict, total=False):
+    ...
+
+
+def generate_parms_dict(data: DataType) -> dict:
```

Comment:
I think `generate_parameters_dict` or `generate_params_dict` would be a better name